lib/Resque.pm

```diff
@@ -97,7 +101,7 @@ has redis => (
     is      => 'ro',
     lazy    => 1,
     coerce  => 1,
-    isa     => 'Sugar::Redis',
+    isa     => 'Sugar::Redis|Object',
```

Comment:
I wonder if [duck_type](https://metacpan.org/pod/Moose::Util::TypeConstraints#duck_type-name-methods) would make more sense for this type.